.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "next/core-web-vitals"
+}
```

Comment:
Should consider adding more here

- Prettier standards
- Auto import ordering
- Basic rulings like no console.logs

checkout these extends

 
    
    "airbnb",

    "airbnb/hooks",
    
    "airbnb-typescript",
    
    "plugin:@typescript-eslint/recommended",
    
    "plugin:jest/recommended",
    
    "plugin:prettier/recommended"`

Reply:
Using https://nextjs.org/docs/basic-features/eslint base next eslint. As the project grows, it will be a better idea to have our own config for eslint.

Reply:
This eslint file should be figured out at the beginning of the project or you might have a lot of fixing to do once its actually implemented. Using some of the most popular standard extends is recommended to get it started. We can build on it from that